src/website/app/demos/Radio/examples/radio/importSyntax.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+export default {
+  id: 'import-syntax',
+  title: 'Import Syntax',
+  description: `\`\`\`
+import Radio, { RadioGroup } from 'mineral-ui/Radio';
+\`\`\`
+
+<Callout title="Note">
```

Comment:
I don't think we need to say this 3 times on the page. It's in the header and the callout here, and the When/How to Use.

Reply:
Removed from component primary description - leaving here.